.eslintignore

```diff
@@ -1,2 +1,3 @@
-# components/
-# src/
+src/modules/auth/authReducer.test.ts 
+src/api/authInstance.ts
+src/modules/auth/Login/Login.tsx
```

Comment:
We really need this? If I correctly remember, I recommend don't use src folder in eslintignore file.

Reply:
I fix this and add /* eslint-disable */ in files.